src/Drivers/OpenAI/Client.php

```diff
@@ -14,10 +14,12 @@ class Client
 
     public function __construct(
         public readonly string $apiKey,
+        public readonly string $organization,
         public readonly string $url,
     ) {
         $this->client = Http::withHeaders([
```

Comment:
Why not build the headers array prior to the `withHeaders` call?
Something along the lines of:
```suggestion
$headers = [
    'Authorization' => sprintf('Bearer %s', $this->apiKey),
];

if (!empty($this->organization)) {
    $headers['OpenAI-Organization'] = $this->organization;
}

$this->client = Http::withHeaders($headers)->baseUrl($this->url);
```

Reply:
Noice! 👏🏼 Thanks @yoavf